src/scripts/components/security-rules.js

```diff
@@ -54,10 +55,13 @@ export function initializeSecurityRules() {
 
     const handleEmptyResultSet = () => {
         const searchQuery = inputSearch.value;
-        const activeCategoryFilter = stringToTitleCase(document.querySelector('.controls .active').text);
+        const txt = (document.querySelector('.controls .active')) ? document.querySelector('.controls .active').text : '';
+        const activeCategoryFilter = stringToTitleCase(txt);
```

Comment:
Note: fixing for error `null is not an object (evaluating 'document.querySelector(".controls .active").text')`